arkouda_proxy_server/src/arkouda.rs

```diff
@@ -0,0 +1,26 @@
+use arkouda::arkouda_client::ArkoudaClient;
+use arkouda::ArkoudaRequest;
+
+pub mod arkouda {
+    tonic::include_proto!("arkouda");
+}
+
+#[tokio::main]
+async fn main() -> Result<(), Box<dyn std::error::Error>> {
+    let mut client = ArkoudaClient::connect("http://[::1]:50051").await?;
+
+    let request = tonic::Request::new(ArkoudaRequest {
+        user: "Tonic".into(),
+        token: "Tonic".into(),
+        cmd: "Tonic".into(),
+        format: "Tonic".into(),
+        size: "Tonic".into(),
+        args: "Tonic".into(),
```

Comment:
just so im clear on what's going on, is `"Tonic"` the default value for each of these arguments and then the result from the `await` below gets written into these strings?

Reply:
@pierce314159 good question, that syntax basically maps the incoming protobuf message fields to the Tonic protobuf processing logic

Reply:
ah okay cool!